addsvc/grpc_binding.go

```diff
@@ -20,15 +20,15 @@ type grpcBinding struct{ server.Endpoint }
 // way to manipulate the RPC context, like headers for HTTP. So we don't have
 // a way to transport e.g. Zipkin IDs with the request. TODO.
 func (b grpcBinding) Add(ctx context.Context, req *pb.AddRequest) (*pb.AddReply, error) {
```

Comment:
@peterbourgon where do you envision using the `TraceID` w/ Zipkin? We have the ability to add metadata to the context of gRPC library like this:

```go
import "google.golang.org/grpc/metadata"
// ...

md := metadata.Pairs("TraceID", token)
ctx := context.Background()
ctx = metadata.NewContext(ctx, md)
```

Then we can read it back out like this:

```go
func (b grpcBinding) Add(ctx context.Context, req *pb.AddRequest) (*pb.AddReply, error) {
  md, _ := metadata.FromContext(ctx)
  // access to the TraceID: md["TraceID"]
}
```

https://godoc.org/google.golang.org/grpc/metadata

I can add a PR for this if you like. I'm having a little trouble getting the forked version of the repo running as all the include paths point the mainline. Not sure if there is an easier way to deal with that.